api/bin/chainflip-cli/src/main.rs

```diff
@@ -71,6 +71,7 @@ async fn run_cli() -> Result<()> {
 								params.boost_fee,
 								Default::default(),
 								None,
+								None,
```

Comment:
https://linear.app/chainflip/issue/PRO-1544/add-params-to-swaprequestparams-in-cli